py-polars/polars/expr/expr.py

```diff
@@ -8269,48 +8269,62 @@ def extend_constant(self, value: PythonLiteral | None, n: int) -> Self:
 
         return self._from_pyexpr(self._pyexpr.extend_constant(value, n))
 
-    def value_counts(self, *, multithreaded: bool = False, sort: bool = False) -> Self:
+    @deprecate_renamed_parameter("multithreaded", "parallel", version="0.19.0")
+    def value_counts(self, *, sort: bool = False, parallel: bool = False) -> Self:
```

Comment:
I switched the parameters here. I believe the "what" should come before the "how". i.e. `parallel` won't impact the result, only how it's computed. So these params should come last if possible.